src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_VALUE = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        final int figureParameter = 10;
+        final String color = Color.WHITE.name().toLowerCase();
+        Figure figure = new Circle(figureParameter);
+        figure.setColor(color);
+        return figure;
+    }
+
+    public Figure getRandomFigure() {
+        Figure figure = getFigure(random.nextInt(FIGURE_COUNT));
+        figure.setColor(colorSupplier.getRandomColor());
+        return figure;
+    }
+
+    private Figure getFigure(int figureNumber) {
+        switch (figureNumber) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomSquare();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomIsoscelesTrapezoid();
+            default:
+                return getRandomIsoscelesRightTriangle();
+        }
+    }
+
+    private Figure getRandomCircle() {
+        return new Circle(random.nextInt(MAX_VALUE + 1));
```

Comment:
If you want the maximum value to 11, specify MAX_VALUE as 11 instead of adding it everywhere. Fix everywhere

Reply:
You didn't fix it

Reply:
If I do as suggested, the nextInt method might return 0. Does that make sense?